### PR TITLE
en: update tidb operator overview

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Update tidb operator overview.

This PR cannot be merged until `configure-cluster-using-tidbcluster.md` in `en` is merged and `en/upgrade-a-tidb-cluster.md` is updated.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v1.1 changes for now)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  https://github.com/pingcap/docs-tidb-operator/pull/51
- Other reference link(s):

### CODE REVIEW

File: en/tidb-operator-overview.md [hunk 1]
```diff
@@ -14,22 +14,33 @@ category: reference
 
 ## TiDB Operator architecture
 
-![TiDB Operator Overview](/media/tidb-operator-overview.png)
+![TiDB Operator Overview](/media/tidb-operator-overview-1.1.png)
 
-`TidbCluster` is a custom resource defined by CRD (`CustomResourceDefinition`) and is used to describe the desired state of the TiDB cluster. The following components are responsible for the orchestration and scheduling logic in a TiDB cluster:
+`TidbCluster`, `TidbMonitor`, `TidbInitializer`, `Backup`, `Restore`, `BackupSchedule`, and `TidbClusterAutoScaler`is a custom resource defined by CRD (`CustomResourceDefinition`).
```
Comment (ran-huang) [inline en/tidb-operator-overview.md]:
```suggestion
`TidbCluster`, `TidbMonitor`, `TidbInitializer`, `Backup`, `Restore`, `BackupSchedule`, and `TidbClusterAutoScaler` are custom resources defined by CRD (`CustomResourceDefinition`).
```

File: en/tidb-operator-overview.md [hunk 2]
```diff
@@ -14,22 +14,33 @@ category: reference
...
+The diagram above is the analysis of the control flow of TiDB Operator. Starting from TiDB Operator v1.1, the TiDB cluster, monitoring, initialization, backup, and other components are deployed and managed using CR. The overall control flow is described as follows:
 
-1. The user creates a `TidbCluster` object and a corresponding series of Kubernetes-native objects through Helm, such as a `CronJob` that performs scheduled backups;
-2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet` and `Service` objects of PD, TiKV, and TiDB based on the actual state of the cluster;
-3. Kubernetes' native controller creates, updates, and deletes the corresponding `Pod` based on objects such as `StatefulSet`, `Deployment`, and `CronJob`;
+1. The user creates a `TidbCluster` object and other CR objects through kubectl, such as `TidbMonitor`;
+2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet`, `Deployment` and `Service`, and other objects of PD, TiKV, TiDB, Monitor or other components based on the actual state of the cluster;
```
Comment (ran-huang) [inline en/tidb-operator-overview.md]:
```suggestion
2. TiDB Operator watches `TidbCluster` and other related objects, and constantly adjust the `StatefulSet`, `Deployment`, `Service`, and other objects of PD, TiKV, TiDB, Monitor or other components based on the actual state of the cluster;
```

File: en/tidb-operator-overview.md [hunk 3]
```diff
@@ -14,22 +14,33 @@ category: reference
...
+3. Kubernetes' native controller creates, updates, and deletes the corresponding `Pod` based on objects such as `StatefulSet`, `Deployment`, and `Job`;
```
Comment (DanielZhangQD) [inline en/tidb-operator-overview.md]:
```suggestion
3. Kubernetes' native controllers create, update, or delete the corresponding `Pod` based on objects such as `StatefulSet`, `Deployment`, and `Job`;
```

File: en/tidb-operator-overview.md (hunk 2 above)
Comment (DanielZhangQD) [reply en/tidb-operator-overview.md]: "the `StatefulSet`, `Deployment`, `Service`, and other objects" or "the `StatefulSet`, `Deployment`, `Service` and other objects"?

File: en/tidb-operator-overview.md [hunk 4]
```diff
@@ -14,22 +14,33 @@ category: reference
...
+* `TidbMonitor` describes the monitoring components of the TiDB cluster.
+* `TidbInitializer` describes the desired initialization Job of the TiDB cluster.
+* `Backup` describes the desired backup of the TiDB cluster.
+* `Restore` describes the desired restoration of the TiDB cluster.
+* `BackupSchedule` describes the scheduled backup of the TiDB cluster.
+* `TidbClusterAutoScaler` describes the automatic scaling of the TiDB cluster.
+
+The following components are responsible for the orchestration and scheduling logic in a TiDB cluster:
+
+* `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state and complete the corresponding control logic according to other CR;
```
Comment (DanielZhangQD) [inline en/tidb-operator-overview.md]:
```suggestion
* `tidb-controller-manager` is a set of custom controllers in Kubernetes. These controllers constantly compare the desired state recorded in the `TidbCluster` object with the actual state of the TiDB cluster. They adjust the resources in Kubernetes to drive the TiDB cluster to meet the desired state and complete the corresponding control logic according to other CRs;
```

File: en/tidb-operator-overview.md (hunk 2 above)
Comment (ran-huang) [reply en/tidb-operator-overview.md]: I think the former is right. When you describe a series of items, put a comma before the final `and` or `or`. You can refer to the [Google Style Guide | commas](https://developers.google.cn/style/commas#serial-commas_1) 😀

Comment (DanielZhangQD) [reply en/tidb-operator-overview.md]: OK.

**Comment by TomShawn** [comment]
@DanielZhangQD @ran-huang Comments addressed, PTAL, thanks!

File: en/tidb-operator-overview.md [hunk 5]
```diff
@@ -57,15 +68,16 @@ TiDB Operator provides several ways to deploy TiDB clusters in Kubernetes:
...
 
 After the deployment is complete, see the following documents to use, operate, and maintain TiDB clusters in Kubernetes:
 
 + [Access the TiDB Cluster](access-tidb.md)
 + [Scale TiDB Cluster](scale-a-tidb-cluster.md)
 + [Upgrade TiDB Cluster](upgrade-a-tidb-cluster.md#upgrade-the-version-of-tidb-cluster)
 + [Change the Configuration of TiDB Cluster](upgrade-a-tidb-cluster.md#change-the-configuration-of-tidb-cluster)
-+ [Backup and Restoration Based on Helm Charts](backup-and-restore-using-helm-charts.md)
++ [Back up a TiDB Cluster](backup-to-s3.md)
++ [Restore a TiDB Cluster](restore-from-s3.md)
```
Comment (ran-huang) [inline en/tidb-operator-overview.md]: `backup-to-s3.md` and `backup-to-aws-s3-using-br.md` are two separate files. The latter is still under review in https://github.com/pingcap/docs-tidb-operator/pull/45, so the CI check might report deadlink on this one.
```suggestion
+ [Back up a TiDB Cluster](backup-to-aws-s3-using-br.md)
+ [Restore a TiDB Cluster](restore-from-aws-s3-using-br.md)
```

**Pull request merged**